src/tests/homework.spec.js

```diff
@@ -0,0 +1,221 @@
+// Homework 1 
+
+import { test, expect } from "@playwright/test";
+
+//return locator objects
+
+async function getNameFieldLocator(page) {
+    return page.locator("#name");
+}
+async function getEmailFieldLocator(page){
+    return page.getByLabel("Email");  
+}
+
+async function getPasswordFieldLocator(page) {
+    return page.getByLabel("Heslo");
+}
+async function getControlPasswordFieldLocator(page) {
+    return page.getByLabel("Kontrola hesla");
+}
+async function getRegistraceButtonLocator (page) {
+    return page.getByRole('button', {name: 'Zaregistrovat'});
+}
+
+async function getCurrentUserLocator(page) 
```

Comment:
Špatné formátování této funkce a i dalších, kudrnatá závorka se píše na stejném řádku jako název funkce, tedy následovně:
async function getCurrentUserLocator(page) {
    const currentUser = page
        .locator(".navbar-right")
        .locator("strong");
    return currentUser;
}